### DISCUSSION TITLE
Anzeige der Daten in HA_ es werden nur historische Daten bis Status quo angeziegt

### DISCUSSION THREAD

**Discussion by magnitka1** [mampfes/ha_epex_spot#182] [Q&A]
Hallo zusammen,
eine kurze Frage: wieso werden im Diagramm nur die Daten der aktuellen Stunde und die bisherigen Daten, jedoch nicht d+1 Daten angezeigt? Offenbar dokumentiert Sensor nur die aktuelle Stunde und interpretiert die Daten für die Zukunft nicht, obwohl diese verfügbar sind.
VG
![image](https://github.com/user-attachments/assets/0defb556-b83f-4b37-b4d3-a05d46281c35)

**Comment by rafichris** [comment]
Hi,

I think the data will be published usually every day after 13:00 (1pm) for the next 24h.
In my dashboard I'm using the [apexcharts-card](https://github.com/RomRider/apexcharts-card) which looks pretty nice.
![image](https://github.com/user-attachments/assets/b40b84d9-8c3c-4ff4-81f8-22eef221741f)

Here's the code for the card:

type: custom:apexcharts-card
header:
  show: true
  title: Electricity Prices
graph_span: 48h
span:
  start: day
now:
  show: true
  label: Now
series:
  - entity: sensor.epex_spot_data_net_price
    name: Electricity Price
    type: column
    extend_to: end
    data_generator: |
      return entity.attributes.data.map((entry) => {
        return [new Date(entry.start_time), entry.price_ct_per_kwh];
      });

I've also cross checked it against my manual node red implementation. 
Result: same result.

Good luck,
BR Chris
